vipps-report-api-faq.md

```diff
@@ -142,7 +154,7 @@ Later (we can not give an ETA, but will update here):
 The merchant's own API keys give full access to the Report API.
 
 If the merchant uses an integration partner (see
-[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner#partner-types)),
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
```

Comment:
```suggestion
[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner#partner-types)),
```

Reply:
You can't add that slash there, Christian. It is one of the ones that gives a 404 :-0